pkg/build/build.go

```diff
@@ -320,7 +323,7 @@ func BuildS2I(image, context, builderImage string) error {
 	}
 
 	log.Infof("Starting build for %q", image)
-	cmd := []string{"oc", "start-build", image, "--from-dir=" + context, "-F"}
+	cmd := []string{"oc", "start-build", image, "--from-dir=" + context, "-F", "--namespace", namespace}
```

Comment:
I'm not sure that this will work if a namespace is not set.

Reply:
@kadel it works:

```
# ./kedge build --s2i -i s2i -b centos/python-35-centos7
imagestream "s2i" created
buildconfig "s2i" created
INFO[0012] Starting build for "s2i"
build "s2i-1" started

Pushing image 172.30.1.1:5000/myproject/s2i:latest ...
Pushed 0/10 layers, 0% complete
Pushed 1/10 layers, 10% complete
Pushed 2/10 layers, 21% complete
Pushed 3/10 layers, 32% complete
Pushed 4/10 layers, 42% complete
Pushed 5/10 layers, 55% complete
Pushed 6/10 layers, 65% complete
Pushed 7/10 layers, 87% complete
Pushed 8/10 layers, 90% complete
Pushed 9/10 layers, 95% complete
Pushed 10/10 layers, 100% complete
Push successful


# oc get all
NAME      TYPE      FROM      LATEST
bc/s2i    Source              1

NAME           TYPE      FROM             STATUS     STARTED         DURATION
builds/s2i-1   Source    Binary@e47a415   Complete   5 minutes ago   4m24s

NAME      DOCKER REPO                     TAGS      UPDATED
is/s2i    172.30.1.1:5000/myproject/s2i   latest    43 seconds ago

NAME             READY     STATUS      RESTARTS   AGE
po/s2i-1-build   0/1       Completed   0          5m
```